pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: layout.pug
+navigationTitle: Konvoy 2.1.4 Release Notes
+title: Konvoy 2.1.4 Release Notes
+menuWeight: 50
+excerpt: View release-specific information for Konvoy 2.1.4
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Konvoy&reg; version 2.1.4 was released on October dd, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+To get started with Konvoy, [download](../../download/) and [install](../../choose-infrastructure/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+## Fixes and Improvements
+
+<Add or remove>
+
+## Component updates
+
+The following services and service components are upgraded to the listed version:
+
+- centralized-grafana: 18.1.3
```

Comment:
Do these all need to be updated? @bcmendoza can help.

Reply:
Newly generated table for this release, please copy and paste the table into this section: https://github.com/mesosphere/dkp-infra-tools/blob/6c64a8dda91cd9865e8084e8c9eab802e7b3cf78/automation/cmd/report/hack/v2.1.4-rc.2.md?plain=1